src/ovs/aca_arp_responder.cpp

```diff
@@ -274,14 +275,16 @@ void ACA_ARP_Responder::_parse_arp_request(uint32_t in_port, vlan_message *vlanm
   if (!_arp_db.find(stData, current_arp_data)) {
     ACA_LOG_DEBUG("ARP entry does not exist! (ip = %s and vlan id = %u)\n",
                   stData.ipv4_address.c_str(), stData.vlan_id);
-    arp_xmit(in_port, vlanmsg, arpmsg, 0);
-    return;
+    // arp_xmit(in_port, vlanmsg, arpmsg, 0);
```

Comment:
we may want to remove the commented out line and add comment on what's going to happen when we return ENOTSUP.

Reply:
fixed